providers/openshift.md

```diff
@@ -184,31 +108,7 @@ vm openshift on your development machine) but is an OpenShift v4 cluster.
     oc login -u kubeadmin -p KUBEADMING_PASSWORD https://api.crc.testing:6443
     ```
 
-5. Now you can setup the management-infra/management-admin service account for use with ManageIQ
-    ```console
-    git clone https://gist.github.com/e2fac8be87ea0e9f429b6f5d75e02176 /tmp/manageiq_crc
```

Comment:
This gist was quite old and it is better to point to our existing openshift documentation since that is kept up to date